scripts/utils.js

```diff
@@ -10,10 +10,8 @@ export const targets = fs.readdirSync('packages').filter(f => {
     return false
   }
   const pkg = require(`../packages/${f}/package.json`)
-  if (pkg.private && !pkg.buildOptions) {
```

Comment:
The original code shows the intention more clearly IMO.

Reply:
Yes, this way the logic will be clearer